aten/src/ATen/templates/CodegenUnboxingWrappers.cpp

```diff
@@ -0,0 +1,42 @@
+#include "torch/csrc/jit/runtime/operator.h"
```

Comment:
nit: this file probably shouldn't go in the `aten` subfolder, since it's part of the `torch` namespace. We have a few other codegen'd template files that fit in the `torch` namespace under `tools/autograd/templates/`, so maybe that's a (slightly) better home for this file.